static/js/chart/draw_map_utils.ts

```diff
@@ -254,5 +254,5 @@ export function getPlacePathId(placeDcid: string): string {
   if (_.isEmpty(placeDcid)) {
     return "";
   }
-  return placeDcid.replace("/", "-");
+  return placeDcid.replace(/\//g, "-");
```

Comment:
use `.replaceAll` instead of a regex (a slightly cheaper operation)

Reply:
done